src/raftstore/store/peer.rs

```diff
@@ -699,7 +683,7 @@ impl Peer {
             }
             vec![]
         } else {
-            self.handle_raft_commit_entries(&ready.committed_entries)
+            self.handle_raft_commit_entries(ready.committed_entries.take().unwrap())
```

Comment:
I think we should rename `handle_raft_commit_entries` to `handle_raft_committed_entries`.